docs/read-the-docs/source/workceptor.rst

```diff
@@ -0,0 +1,182 @@
+Workceptor
+==========
+
+Workceptor is a component of receptor that handles units of work.
+
+``work-command`` defines a type of work that can run on the node.
+
+foo.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+        id: foo
+
+    - log-level:
+        level: Debug
+
+    - tcp-listener:
+        port: 2222
+
+    - control-service:
+        service: control
+        filename: /tmp/foo.sock
+
+    - work-command:
+        workType: echosleep
+        command: bash
+        params:  "-c \"for i in {1..5}; do echo $i; done\""
+
+bar.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+        id: bar
+
+    - log-level:
+        level: Debug
+
+    - tcp-peer:
+        address: localhost:2222
+
+    - control-service:
+        service: control
+
+    - work-command:
+        worktype: echosleep
+        command: bash
+        params:  "-c \"for i in {1..10}; do echo $i; sleep 1; done\""
+
+
+Configuring work commands
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``worktype`` User-defined name to give this work definition
```

Comment:
Are there any restrictions on the characters that can be used in a worktype name? Restricted to alphanumeric + underscores / dashes?